tests/SAML2/XML/saml/AttributeValueTest.php

```diff
@@ -95,9 +95,8 @@ public function testCreateAttributeFromDOMElement(): void
     public function testSerialize(): void
     {
         $av1 = new AttributeValue("Aap:noot:mies");
-        $ser = $av1->serialize();
         $av2 = new AttributeValue("Wim");
-        $av2->unserialize($ser);
+        $av2 = unserialize(serialize($av1));
 
         $this->assertEquals("Aap:noot:mies", $av2->getString());
```

Comment:
I tried to write these tests a bit simpler. If we already have a DOMDocument in the test class, we can just load it with `fromXML()`, serialize, unserialize, and then compare the string value with the result of calling `saveXML()` on the document.

Reply:
I think it has some ties to the rewrite-attributes PR, so we should probably fix it there..